ollama/runtime/src/main/java/io/quarkiverse/langchain4j/ollama/runtime/OllamaRecorder.java

```diff
@@ -17,11 +17,18 @@ public class OllamaRecorder {
     public Supplier<?> chatModel(Langchain4jOllamaConfig runtimeConfig, String modelName) {
         Langchain4jOllamaConfig.OllamaConfig ollamaConfig = correspondingOllamaConfig(runtimeConfig, modelName);
         ChatModelConfig chatModelConfig = ollamaConfig.chatModel();
+
+        int seed=null;
+        if(null!=chatModelConfig.seed()){
```

Comment:
As `chatModelConfig.seed` will not always be an optional, it needs to be handled accordingly :)